src/many/Cargo.toml

```diff
@@ -15,7 +15,7 @@ base32 = "0.4.0"
 base64 = "0.13.0"
 cbor-diag = "0.1.8"
 crc-any = "2.4.0"
-cryptoki = "0.3"
+cryptoki = { version = "0.3", features = ["generate-bindings"] }
```

Comment:
Reason for enabling this?

Reply:
Doesn't build on ARM CPUs. Level2 archs and stuff.

Reply:
I _could_ special case it for arm archs if you want. Does that cost you a lot of compile time?